src/classes/UTIL_UnitTestData_TEST.cls

```diff
@@ -436,14 +573,21 @@ public class UTIL_UnitTestData_TEST {
         return buildUser(buildUniqueLastName(), profileName);
     }
 
+    /**
+      * @description Generate a unique string to append to indexed fields to make them unique
+      */
+    public static String getUniqueString() {
+        return String.valueOf(Math.abs(Crypto.getRandomLong()));
```

Comment:
This method is intended to generate a completely unique string value to add to a Contact.FirstName, Contact.LastName, Account.Name, etc. In testing, I found that using `getRandomInteger()` could still return a duplicate value if the method was called repeatedly in a loop of 50000 iterations. That did not happen using `getRandomLong()` even in a loop of 100000 iterations.